lib/cdk-project-stack.ts

```diff
@@ -35,6 +36,70 @@ export class CdkProjectStack extends cdk.Stack {
     });
     const apiIntegration = new apigateway.LambdaIntegration(lambdaFunction);
 
+    //Cognito
+    const userPool = new cognito.UserPool(this, 'userPool', {
+      selfSignUpEnabled: true,
+      userVerification: {
+        emailSubject: 'Verify your email!',
+        emailBody: 'Hello, Thanks for signing up! {##Verify Email##}',
+        emailStyle: cognito.VerificationEmailStyle.LINK
+      },
+      signInAliases: {
+        username: true,
+        email: true
+      },
+      standardAttributes: {
+        email: {
+          required: true
+        }
+      }
+    });
+
+    const userPoolIdentityProviderGoogle = new cognito.UserPoolIdentityProviderGoogle(this, 'userPoolGoogle', {
+      userPool: userPool,
+      //OAuth 2.0 Client ID/Secret generated in my GCP account
```

Comment:
Is this comment from some copied code?

Reply:
No I just added that myself.  Probably didn't need to.